doozer/doozerlib/cli/fbc.py

```
@@ -721,17 +721,38 @@ async def run(self):
         ]
 
         results = await asyncio.gather(*tasks, return_exceptions=True)
+        successful_nvrs = []
         failed_tasks = []
+        errors = []
+
         for dgk, result in zip(dgk_bundle_builds, results):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)